### PR TITLE
Fix size changing for banner without image

### PR DESCRIPTION
**Pull request by djnunez-aot**
*Issue #504 :*
https://github.com/bcgov/met-public/issues/504

-Adding small screen header change for responsiveness

-Fixed the size of the description changing

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

File: met-web/src/components/engagement/banner/BannerWithoutImage.tsx [hunk 1]
```diff
@@ -54,8 +55,8 @@ const BannerWithoutImage = ({ savedEngagement }: BannerProps) => {
                         m={{ lg: '3em 5em 0 3em', md: '3em', sm: '1em' }}
                         rowSpacing={2}
                     >
-                        <Grid item xs={12}>
-                            <Typography variant="h3">{name}</Typography>
+                        <Grid item xs={12} sx={{ maxHeight: '20em', overflowY: 'auto', overflowX: 'auto' }}>
+                            <Typography variant={isSmallScreen ? 'h3' : 'h4'}>{name}</Typography>
                             <Editor editorState={getEditorState(rich_description)} readOnly={true} toolbarHidden />
```
Comment (tiagograf-AOT) [inline met-web/src/components/engagement/banner/BannerWithoutImage.tsx:60]: I think this bug is related to the description extending out of the bounds, likely implemented in the banner with image

Comment (djnunez-aot) [reply met-web/src/components/engagement/banner/BannerWithoutImage.tsx:60]: yea that's actually what we did in the banner with the image so I just copied the solution

Comment (tiagograf-AOT) [reply met-web/src/components/engagement/banner/BannerWithoutImage.tsx:60]: Cool, sorry I didn’t see the scroll attributes

**Review by tiagograf-AOT — APPROVED** [review]

File: met-web/src/components/engagement/banner/BannerWithoutImage.tsx (hunk 1 above)
Comment (djnunez-aot) [reply met-web/src/components/engagement/banner/BannerWithoutImage.tsx:60]: All good :)

**Review by jadmsaadaot — APPROVED** [review]

**Pull request merged**